src/components/MoviesList/MoviesList.tsx

```diff
@@ -4,12 +4,22 @@ import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  query: string;
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
+function isNeeded(movieInfo: string, query: string) {
```

Comment:
stick to descriptive naming
```suggestion
function doesMoviePropertyMatchQuery(movieProperty: string) {
```